src/ZendCodingStandard/ruleset.xml

```diff
@@ -2,178 +2,47 @@
-    <rule ref="WebimpressCodingStandard.NamingConventions.Exception"/>
-    <!-- Interface must have *Interface suffix -->
-    <rule ref="WebimpressCodingStandard.NamingConventions.Interface"/>
-    <!-- Trait must have *Traif suffix -->
-    <rule ref="WebimpressCodingStandard.NamingConventions.Trait"/>
-    <!-- Require camelCase variable names -->
-    <rule ref="WebimpressCodingStandard.NamingConventions.ValidVariableName"/>
-
-    <!-- Use the correct class name, i.e. DateTime not Datetime, etc -->
-    <rule ref="WebimpressCodingStandard.PHP.CorrectClassNameCase"/>
-
-    <!-- Only one blank line allowed -->
-    <rule ref="WebimpressCodingStandard.WhiteSpace.BlankLine"/>
-
-    <!-- Indents formatting -->
-    <rule ref="WebimpressCodingStandard.WhiteSpace.ScopeIndent">
-        <properties>
-            <property name="alignObjectOperators" value="false"></property>
-        </properties>
+    <!-- Code MUST follow all rules outlined in PSR-1. -->
+    <rule ref="PSR12">
+        <exclude name="Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen"/>
+        <exclude name="Squiz.WhiteSpace.ControlStructureSpacing.SpacingBeforeClose"/>
+        <!-- checked by WebimpressCodingStandard.WhiteSpace.BraceBlankLine -->
     </rule>
 
-    <!-- Detects for-loops that can be simplified to a while-loop -->
-    <rule ref="Generic.CodeAnalysis.ForLoopShouldBeWhileLoop"/>
-    <!-- Detects unconditional if- and elseif-statements -->
-    <rule ref="Generic.CodeAnalysis.UnconditionalIfStatement"/>
-    <!-- Disallow spaces before/after incrementation/decrementation operators -->
-    <rule ref="Generic.WhiteSpace.IncrementDecrementSpacing"/>
-
-    <!-- Forbid inline comments after closing curly bracket, like // end if etc -->
-    <rule ref="WebimpressCodingStandard.Commenting.NoInlineCommentAfterCurlyClose"/>
-    <!-- Use correct case of default PHPDocs and PHPUnit tags @var not @VAR etc -->
-    <rule ref="WebimpressCodingStandard.Commenting.TagCase"/>
-    <!-- Heredoc/nowdoc tag uppercase without spaces -->
-    <rule ref="WebimpressCodingStandard.Formatting.Heredoc"/>
-
-    <!-- Forbid space after splat operator -->
-    <rule ref="WebimpressCodingStandard.Formatting.NoSpaceAfterSplat"/>
-    <!-- Forbid reference operator before new, and formatting of the reference operator -->
-    <rule ref="WebimpressCodingStandard.Formatting.Reference"/>
-    <!-- Use ::class instead of class string reference -->
-    <rule ref="WebimpressCodingStandard.Formatting.StringClassReference"/>
-    <!-- Single line between methods in class -->
-    <rule ref="WebimpressCodingStandard.Methods.LineAfter"/>
-
-    <!-- Forbid space before comma, require single space after comma -->
-    <rule ref="WebimpressCodingStandard.WhiteSpace.CommaSpacing"/>
+    <!-- The term 'StudlyCaps' in PSR-1 MUST be interpreted as PascalCase where
+         the first letter of each word is capitalized including the very first
+         letter. -->
```

Comment:
It's an addition to `Code MUST follow all rules outlined in PSR-1.` I'm guessing that saying `Class names MUST be declared in StudlyCaps.` wasn't clear enough. This could be removed but since the PSR-12 text is followed I left it there.